libs/dot-rules/src/lib/services/ServerSideFieldModel.ts

```diff
@@ -6,6 +6,11 @@ import { LoggerService } from '@dotcms/dotcms-js';
 import { CustomValidators } from './validation/CustomValidators';
 
 export class ServerSideFieldModel extends CwModel {
+    private static readonly QUOTE_ALLOW_HEADER_KEYS: Array<string> = [
+        'Set Request Attribute',
+        'Set Response Header',
+        'Set Session Attribute'
+    ];
```

Comment:
Can't we just make the input to support single quotes always?

Reply:
Done.